js/debug-bar.dev.js

```diff
@@ -52,7 +55,7 @@ var wpDebugBar;
 				} );
 			},
 			visibility: function visibility( show ) {
-				show = typeof show == 'undefined' ? ! api.isVisible() : show;
+				show = typeof 'undefined' === show ? ! api.isVisible() : show;
```

Comment:
Thanks for catching that! I was clearing doing too much too quickly ;-) I've redone the commit to include the fix for this.